### DISCUSSION TITLE
Error searching messages

### DISCUSSION THREAD

**Discussion by Mishelovka** [victornpb/undiscord#582] [1. Questions & Answers]
I got a problem when i trying to delete my messages in DM

Error searching messages, API responded with status 401!
	{"message":"401: Unauthorized","code":0}
CoreException	{}

Does script work for dm?

**Comment by Mishelovka** [comment]
After some manipulation i got this


Error searching messages, API responded with status 400!
	{"message":"Invalid form body text","code":50035,"errors":{"max_id":{"_errors":[{"code":"NUMBER_TYPE_COERCE","message":"Value «here i have my mail» isnt snowflake."}]}}}
CoreException	{}

**Comment by SD-Chargers** [comment]
You're using Chrome ( or any other Chrome based webbrowser ) ?

I had the same problem, found that the application was using my stored "password manager" infos as API tokens.

Deleted all saved password in the browser for discord.com, and - suprise - it worked again.

Try that and lemme know
